aut_docs/Installation_Guide_WindowsOS.md

```diff
@@ -0,0 +1,235 @@
+## Installation Guide for Windows OS
+
+### A one-time setup:
+
+- Download Python (Windows x86-64 executable installer version 3.6.5) 
+    ```bash
+      [Python 3.6.5](https://www.python.org/downloads/release/python-365)
+    ```
+  Note: Don't forget to check **Add to path** while installing.
+
+- Check the python version 
+    ```bash
+      python --version
+    ```
+
+- Check if pip (The python package installer) is already installed by running
+    ```bash
+      pip --version
+    ```
+- If installed then upgrade it using the command
+    ```bash
+      python -m pip install --upgrade pip
+    ```
+  This will upgrade the pip version.
+
+- Install virtual environment
+    ```bash
+      pip install virtualenv
+    ```
+- Download Git 
+    ```bash
+       [Git](https://git-scm.com/download/win)
+    ```
+     Note: While installing select the option **Use Git and optional Unix Tools from the Windows Command Prompt** for using Git from the command prompt
+ 
+- Clone the VMS project
+    ```bash
+      git clone https://github.com/systers/vms.git
+    ```
+
+- Create python3.6 virtual environment
+    ```bash
+      virtualenv -p python3.6 venv
+    ```
+    
+### To start a testing session:
+
+- Activate virtual environment
+    ```bash
+      .\venv\Scripts\activate
+    ```
+    
+- Change directory to VMS code
+    ```bash
+      cd vms\vms
+    ```
+
+- Install all python dependencies
+    ```bash
+      pip install -r requirements.txt
+    ```
+
+### Install PostgreSQL
+
+- If you are installing and running the project on your local machine, then you will need to download and install PostgreSQL (Remember the password that you have entered during the installation process):
+
+[PostgreSQL](http://www.postgresql.org/download/) (version >= 9.3.4)
+
+  Note: Add bin & lib to path, Example: **C:\Program Files\PostgreSQl\9.5\bin** & **C:\Program Files\PostgreSQl\9.5\lib**.
+
+- Download Firefox web browser
+[Firefox](https://www.mozilla.org/en-US/firefox/new/) 
+    
+### Setup PostgreSQL
+
+- To start the postgreSQL Server there are 3 ways
+
+## Command Prompt
+    ```bash
```

Comment:
I can see the word `bash` here after i previewed this doc. Can you please fix this?

Reply:
I will change bash to cmd. I guess that will work fine.

Reply:
I suppose you don't either need to write 'cmd' or 'bash'. The markdown (```...```) itself shows that code needs to be written in the cmd console.

Reply:
ya that's right but as for the linux os it was written bash so I thought that writing cmd would be easy for the newcomers who view the guide .md file